Mathlib/Algebra/Category/ModuleCat/Abelian.lean

```diff
@@ -128,4 +129,30 @@ theorem exact_iff : Exact f g ↔ LinearMap.range f = LinearMap.ker g := by
 set_option linter.uppercaseLean3 false in
 #align Module.exact_iff ModuleCat.exact_iff
 
+theorem shortComplex_exact (H : LinearMap.range f = LinearMap.ker g) :
```

Comment:
It seems to me that these lemmas are already in the file `Algebra.Homology.ShortComplex.ModuleCat`.

Reply:
(If this is needed, note also that in #14520, I introduce a lemma which relates `Function.Exact` and  `ShortComplex.Exact` in the category of modules.)

Reply:
Thanks! That will save may rewrites `LinearMap.exact_iff`